swal-forms.js

```diff
@@ -46,10 +46,13 @@
     addWayToGetFormValuesInDoneFunction: function(swalArgs) {
       var swalFormInstance = this
       var doneFunction = swalArgs[1]
-      swalArgs[1] = function() {
+      swalArgs[1] = function(isConfirm) {
```

Comment:
I believe this is the way I said before. You can check this by open your console(f12 in google chrome or firefox). then:
`swal({title: "teste"},function(){alert("hello")})`
Press Esc key.
Now type this:
`swal({title: "teste"},function(someArgument){alert("hello")})`
Press Esc key again